content/chainguard/chainguard-images/troubleshooting/debugging-distroless-images.md

```diff
@@ -45,21 +45,21 @@ For example, the following table shows a comparison between the `dev` variants o
 
 You can find similar detailed package information for all [Chainguard Containers](https://images.chainguard.dev) in their respective image details pages under the SBOM section.
 
-Once you have changed your Dockerfile base image to use a `-dev` variant, you can overwrite the entry point command to get a shell on the container:
+Once you have changed your Dockerfile base image to use a development variant, you can overwrite the entry point command to get a shell on the container:
 
 ```shell
 docker run -it --entrypoint /bin/sh cgr.dev/chainguard/php:latest-dev
 ```
 
 Having a package manager and the ability to log into the image to debug any issues is very important at development time, but becomes unnecessary (and less safe) when talking about production environments. That's why we recommend using a distroless variant for production workloads.
 
-### Chainguard Containers in Production
-Although the `-dev` image variants have similar security features as their distroless versions, such as complete SBOMs and signatures, they feature additional software that is typically not necessary in production environments. The general recommendation is to use the `-dev` variants to build the application and then copy all application artifacts into a distroless image, which will result in a final container image that has a minimal attack surface and won't allow package installations or logins.
+### Why You Should Avoid Using Dev Images in Production
+Although the development image variants have similar security features as their distroless versions, such as complete SBOMs and signatures, they feature additional software that is typically not necessary in production environments. The general recommendation is to use the Development variants only to build the application and then copy all application artifacts into a distroless image, which will result in a final container image that has a minimal attack surface and won't allow package installations or logins.
```

Comment:
Why the use of title case for "Development" here? Does PMM have guidance on that?

"only" is a heavy handed as -dev images are suitable for production.

```suggestion
Although the development image variants have similar security features as their distroless versions, such as complete SBOMs and signatures, they feature additional software that is typically not necessary in production environments. The general recommendation is to use the Development variants to build the application and then copy all application artifacts into a distroless image, which will result in a final container image that has a minimal attack surface and won't allow package installations or logins.
```